addOns/automation/src/main/java/org/zaproxy/addon/automation/gui/AutomationPanel.java

```diff
@@ -594,6 +594,7 @@ public void mouseClicked(MouseEvent me) {
                                 if (userObj instanceof AutomationEnvironment) {
                                     ((AutomationEnvironment) userObj).showDialog();
                                 } else if (userObj instanceof AutomationJob) {
+                                    ((AutomationJob) userObj).setJobData(null);
```

Comment:
This makes me a bit nervous - have you checked it doesnt negatively impact the other job dialogs?

Reply:
Tested loading and modification, all changes were applied after save, even if loaded via yaml